src/stopping_solve.jl

```diff
@@ -0,0 +1,97 @@
+struct NNStopping{C,O,S,E} <: NeuralNetDiffEqAlgorithm
+    chain::C
+    opt::O
+    sdealg::S
+    ensemblealg::E
+end
+NNStopping(chain  ; opt=Flux.ADAM(0.1) , sdealg = EM() , ensemblealg = EnsembleThreads()) = NNStopping(chain , opt , sdealg , ensemblealg)
+
+function DiffEqBase.solve(
+    prob::OptimalStoppingProblem,
+    alg::NeuralNetDiffEqAlgorithm;
+    abstol = 1f-6,
+    verbose = false,
+    maxiters = 300,
+    trajectories = 1000,
+    save_everystep = false,
+    dt,
+    kwargs...
+    )
+
+    tspan = prob.tspan
+    sigma = prob.sigma
+    μ = prob.mu
+    g = prob.g
+    u0 = prob.u0
+    ts = tspan[1]:dt:tspan[2]
+    N = size(ts)[1]
```

Comment:
there's no reason to define `N` here. It just reduces readability because you have to look it up. Instead, just use `length(ts)` where needed.

Reply:
I will change this.